third_party/packages/flutter_svg/lib/svg.dart

```diff
@@ -446,6 +446,117 @@ class SvgPicture extends StatelessWidget {
+  ///
+  /// The key for caching will be derived from the `assetName`, `package`, and
+  /// `bundle` arguments. When providing assets from another package, the
+  /// `package` argument must be non-null.
+  ///
+  /// Either the [width] and [height] arguments should be specified, or the
+  /// widget should be placed in a context that sets tight layout constraints.
+  /// Otherwise, the image dimensions will change as the image is loaded, which
+  /// will result in ugly layout changes.
+  ///
+  /// If `matchTextDirection` is set to true, the picture will be flipped
+  /// horizontally in [TextDirection.rtl] contexts.
+  ///
+  /// The `allowDrawingOutsideViewBox` parameter should be used with caution -
+  /// if set to true, it will not clip the canvas used internally to the view box,
+  /// meaning the picture may draw beyond the intended area and lead to undefined
+  /// behavior or additional memory overhead.
+  ///
+  /// A custom `placeholderBuilder` can be specified for cases where decoding or
+  /// acquiring data may take a noticeably long time.
+  ///
+  /// The `color` and `colorBlendMode` arguments, if specified, will be used to set a
+  /// [ColorFilter] on any [Paint]s created for this drawing.
+  ///
+  /// ## Assets in packages
+  ///
+  /// To create the widget with an asset from a package, the [package] argument
+  /// must be provided. For instance, suppose a package called `my_icons` has
+  /// `icons/heart.svg.vec`.
+  ///
+  /// Then to display the image, use:
+  ///
+  /// ```dart
+  /// SvgPicture.precompiled('icons/heart.svg.vec', package: 'my_icons')
+  /// ```
+  ///
+  /// Assets used by the package itself should also be displayed using the
+  /// [package] argument as above.
+  ///
+  /// If the desired asset is specified in the `pubspec.yaml` of the package, it
+  /// is bundled automatically with the app. In particular, assets used by the
+  /// package itself must be specified in its `pubspec.yaml`.
+  ///
+  /// A package can also choose to have assets in its 'lib/' folder that are not
+  /// specified in its `pubspec.yaml`. In this case for those images to be
+  /// bundled, the app has to specify which ones to include. For instance a
+  /// package named `fancy_backgrounds` could have:
+  ///
+  /// ```none
+  /// lib/backgrounds/background1.svg.vec
+  /// lib/backgrounds/background2.svg.vec
+  /// lib/backgrounds/background3.svg.vec
+  ///```
+  ///
+  /// To include, say the first image, the `pubspec.yaml` of the app should
+  /// specify it in the assets section:
+  ///
+  /// ```yaml
+  ///  assets:
+  ///    - packages/fancy_backgrounds/backgrounds/background1.svg.vec
+  /// ```
+  ///
+  /// The `lib/` is implied, so it should not be included in the asset path.
+  ///
+  ///
+  /// See also:
+  ///
+  ///  * <https://flutter.io/assets-and-images/>, an introduction to assets in
+  ///    Flutter.
+  ///
+  /// If [excludeFromSemantics] is true, then [semanticsLabel] will be ignored.
```

Comment:
Copied this part from `SvgPicture.asset` (only changed file names from `NAME.svg` to `NAME.svg.vec`).